ml-agents/mlagents/trainers/torch/components/bc/module.py

```diff
@@ -147,14 +147,11 @@ def _update_batch(
                 )
             )
 
-        memories = [
-            ModelUtils.list_to_tensor(mini_batch_demo["memory"][i])
-            for i in range(
-                0, len(mini_batch_demo["memory"]), self.policy.sequence_length
+        memories = []
+        if self.policy.actor_critic.use_lstm:
```

Comment:
You can also use `policy.use_recurrent`, probably more generalizable than calling the `actor_critic`